docs/api.md

```diff
@@ -2352,6 +2352,184 @@ const puppeteer = require('puppeteer');
 })();
 ```
 
+##### Cooperative Intercept Mode and Legacy Intercept Mode
+
+`request.respond`, `request.abort`, and `request.continue` can accept an optional `priority` to activate Cooperative Intercept Mode. In Cooperative Mode, all intercept handlers are guaraneed to run and the interception will be resolved to the highest priority resolution. Here are the rules of Cooperative Mode:
+
+- Async handlers will finish before intercept resolution is finalized
+- The highest priority interception resolution "wins", ie, the interception will ultimately be aborted/responded/continued according to which resolution was given the highest priority.
+- In the event of a tie, `abort` > `respond` > `continue`
+
+For standardization, when specifying a Cooperative Mode priority use `DEFAULT_INTERCEPT_RESOLUTION_PRIORITY` (currently 10) unless you have a clear reason to use a higher priority. This will gracefully prefer `respond` over `continue` and `abort` over `respond`. If you do intentionally want to use a higher priority, `DEFAULT_INTERCEPT_RESOLUTION_PRIORITY + n` is the recommended approach. For example, `continue({}, DEFAULT_INTERCEPT_RESOLUTION_PRIORITY + 4)` rather than `continue({}, 14)`.
```

Comment:
I'm wondering if there is a way to get around carrying the `DEFAULT_INTERCEPT_RESOLUTION_PRIORITY` constant around all the time. I feel like this makes for a rather bumpy experience. WDYT about changing the default value to 0 and allowing both negative and positive priorities or instead fixing it to some given value like 10?